portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,114 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/* Load blog post data once the rest of the page has loaded. */
+let posts = [];
+document.addEventListener('DOMContentLoaded', function() {
+  let p1 = document.createElement("p");
+  p1.appendChild(document.createTextNode('I have a secret love for ' +
+      'bucket balls. Not only am I randomly good at tossing balls in ' +
+      'incrementally spaced buckets, but I\'ve had lots of opportunity to ' +
+      'do so. Some of my fondest memories are tossing softballs back ' +
+      'into the ball bucket from many yards away at the end of practice, ' +
+      'to the delight of my teammates.'));
+  let img1 = document.createElement("img");
+  img1.src = "/images/bucketballs.png";
+  const post1 = [p1, img1];
+
+  let p2 = document.createElement('p');
+  p2.appendChild(document.createTextNode('My fingers are very ' +
+      'double-jointed. I can bend them all downward at the first joint ' +
+      'behind the nail, and my thumbs bend backwards in a hitchhiker\'s ' +
+      'thumb position. This double-jointedness actually makes it a bit ' +
+      'difficult to play the cello, as collapsing joints are a big no-no ' + 
+      'for your left hand.'));
+  const post2 = [p2];
+  posts = [post1, post2];
+});
+
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['I love hot cheetos.', 'Purple is my favorite color.', 
+      'I\'m horrible at remembering song lyrics.', 'I speak some French.'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Pick a random fact.
+  const fact = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const factContainer = document.getElementById('fact-container');
+  factContainer.innerText = fact;
+}
+
+/**
+ * Shows the specified blog post to the user
+ * @param {number} postNum The number of which
+ * blog post to show.
+ */
+function showBlogPost(postNum) {
+  // Find the correct triangle to expand and switch its image.
+  // Also change the onclick function for this button to hide the post.
+  const buttonId = "expandTriangle" + postNum;
```

Comment:
Use single quotes. Here and below.

https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes